src/routes/(app)/admin[[hash=admin_hash]]/arm/role/new/+page.svelte

```diff
@@ -14,7 +14,10 @@
 		Role name
 		<input class="form-input" type="text" name="name" required placeholder="Role name" />
 	</label>
-
+	<label class="checkbox-label">
+		<input class="form-checkbox" type="checkbox" name="hasPOSOptions" />
+		Has POS
```

Comment:
```suggestion
		Has POS access
```

Even better, "Point-of-sale access". I don't know if we already use "POS" elsewhere; in that case, it's fine to use POS.

I won't comment other occurrences of "Has POS" in UI files, but they should all be addressed.